### ISSUE TITLE
Refactor code or spreadsheets to preserve UUID

### ISSUE THREAD

**Issue by shelleydoljack** [sul-dlss-deprecated/folio-migration-tasks#115]
We want to preserve UUID for some reference settings data to make things easier (like pulling/loading circ rules). Here are the settings data circ rules uses that we need to refactor code or add a UUID column to a spreadsheet:

- loan policies
- request policies
- service points
- patron notice policies
- overdue fine policies
- lost item fee policies
- patron group
- inv. material types (the custom ones)
- calendar?

**Comment by shelleydoljack** [comment]
ID and source columns added to material types tsv.

**Comment by shelleydoljack** [comment]
ID column added to patron-groups.tsv. Loaded to folio-dev and uploaded to Folio Ops google drive.

**Comment by shelleydoljack** [comment]
It looks like the circulation policies (loan, request, overdue fines, etc) don't need anything done since the json we pull includes the id and it is included in the post when doing the different load rake tasks. I deleted/loaded the circulation settings policies on folio-dev to get the UUID's in sync with what is on folio-test. I also tested loading the circ rules using a rake task. This seemed to have worked, however, I think since the UUID's for locations and service points are not the same in folio-dev and folio-test, the place where those are listed in the circulation rules shows UUID's instead of resolving to the human-readable version of the location:
![Screen Shot 2022-12-05 at 3 13 17 PM](https://user-images.githubusercontent.com/6343464/205753775-8126a8c9-459b-4806-9bc0-79672ea49331.png)
This could probably be fixed by copying/pasting the circ rules from folio-test to folio-dev, but in the future, if we refresh data either instance, we should be able to use the load_circ_rules rake task.
